noggin/representation/group.py

```diff
@@ -8,8 +8,11 @@ class Group(Representation):
         "description": "description",
         "members": "member_user",
         "sponsors": "membermanager_user",
+        "url": "fasurl",
```

Comment:
Since it's multi-valued, maybe call it `urls`? We've done that for other values.